cypress/integration/pageobjects/account-management/login-page.ts

```diff
@@ -11,6 +11,16 @@ export default class LoginPage{
         cy.visit('ibpworkbench/controller/auth/login');
     }
 
+    getLocalStorage(item:string){
+       return localStorage.getItem(item)
+    }
+    useLocalStorage(){
+        if(this.getLocalStorage('bms.xAuthToken')==null){
+            this.performLogin();        
+        }
+    }
```

Comment:
Maybe we can call them `getToken()` and `useToken()`?

Reply:
Ok noted on this. Thanks!